jenkins/ci.suse.de/templates/cloud-ardana-pipeline-template.yaml

```diff
@@ -184,18 +184,18 @@
           description: |
             The number of OpenStack core services nodes in the generated input model.
 
-            Input model generator scenarios using this parameter is available: mid-scale, std-split.
+            Input model generator scenarios using this parameter is available: mid-scale-kvm, std-split.
 
             NOTE: this parameter is used to generate input models. See the 'scenario_name' parameter about
             using one of the available input model generator scenarios.
 
       - string:
           name: lmm_nodes
-          default: '{lmm_nodes|1}'
+          default: '{lmm_nodes|3}'
```

Comment:
If you change the default, you should also make sure that the `std-split` jobs are updated to keep the existing values.